spec/features/user_can_see_all_books_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe "User logs in" do
```

Comment:
Careful to read your user stories closely. None of them ask for the functionality to be built for a logged in user.